net/log/net_log_util.cc

```diff
@@ -203,11 +203,14 @@ scoped_ptr<base::DictionaryValue> GetNetConstants() {
   {
     base::DictionaryValue* dict = new base::DictionaryValue();
 
+// TODO (Peter Wang): Remove the temp fix for debug building.
```

Comment:
Seems the code below is related with QUIC, you can try to use macro DISABLE_QUIC_SUPPORT to control it.